tutorials/shading/your_first_shader/your_second_spatial_shader.rst

```diff
@@ -83,8 +83,8 @@ a low ``METALLIC`` has a more equal representation of sky color and ``ALBEDO`` c
 .. note:: ``METALLIC`` should be close to ``0`` or ``1`` for proper PBR shading. Only set it between
           them for blending between materials.
 
-Water is not a metal, so we will set it's ``METALLIC`` property to ``0.0``. But, water is also highly
-reflective, so we will set it's ``ROUGHNESS`` property to by quite low as well.
+Water is not a metal, so we will set its ``METALLIC`` property to ``0.0``. But, water is also highly
```

Comment:
Can you also remove that , after "But"?

Reply:
Hmm, why?

Reply:
Because "But," at the front of a sentence should not be used unless it is followed by a prepositional phrase.